src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -0,0 +1,72 @@
+package reposense.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileUtilTest {
+
+    private static final Path REPORT_DIRECTORY_ABSOLUTE = new File(FileUtilTest.class.getClassLoader()
+            .getResource("reposense-report-test").getFile()).toPath().toAbsolutePath();
+    private static final Path OUTPUT_DIRECTORY_ABSOLUTE = new File(FileUtilTest.class.getClassLoader()
+            .getResource("output").getFile()).toPath().toAbsolutePath();
+    private static final Path ARCHIVE_ZIP_PATH = Paths.get(OUTPUT_DIRECTORY_ABSOLUTE.toString(),
+            FileUtil.ZIP_FILE);
+    private static final Path TEMPLATE_ZIP_PATH = new File(FileUtilTest.class.getClassLoader()
+            .getResource("output/template.zip").getFile()).toPath().toAbsolutePath();
+    private static final Path TEMPLATE_DIRECTORY_ABSOLUTE = Paths.get(OUTPUT_DIRECTORY_ABSOLUTE.toString(), "template");
+
+    @Test
+    public void zip_validLocation_success() throws IOException {
+        FileUtil.zip(REPORT_DIRECTORY_ABSOLUTE, OUTPUT_DIRECTORY_ABSOLUTE, ".json");
+        Assert.assertTrue(Files.exists(ARCHIVE_ZIP_PATH));
+        Assert.assertTrue(Files.size(ARCHIVE_ZIP_PATH) > 0);
+    }
+
+    @Test
+    public void zip_invalidLocation_fail() {
+        Path invalidZipLocation = Paths.get("./zipDoesNotExist.zip");
+        FileUtil.zip(invalidZipLocation, ".json");
+        Assert.assertFalse(Files.exists(ARCHIVE_ZIP_PATH));
+    }
+
+    @Test
+    public void zip_validFileType_success() throws IOException {
+        FileUtil.zip(OUTPUT_DIRECTORY_ABSOLUTE, ".csv");
+        Assert.assertTrue(Files.exists(ARCHIVE_ZIP_PATH));
+        Assert.assertTrue(Files.size(ARCHIVE_ZIP_PATH) > 0);
+    }
+
+    @Test
+    public void zip_invalidFileType_fail() {
+        FileUtil.zip(REPORT_DIRECTORY_ABSOLUTE, ".pdf");
+        Assert.assertFalse(Files.exists(ARCHIVE_ZIP_PATH));
```

Comment:
This test doesn't actually fail, but instead gives an empty `zip` file right? The file type also isn't invalid, just that its non-existent in the folder to zip. Also, add a comment that the 'empty' `zip` file would return `false` for the `Files.exist(Path)` here, since its not intuitive.